app/src/main/res/layout/activity_person_details.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:paddingLeft="@dimen/activity_horizontal_margin"
+    android:paddingRight="@dimen/activity_horizontal_margin"
+    android:paddingTop="@dimen/activity_vertical_margin"
+    android:paddingBottom="@dimen/activity_vertical_margin"
+    tools:context="org.familysearch.sampleapp.activity.PersonDetailsActivity"
+    android:orientation="vertical">
+
+    <ImageView
+        android:id="@+id/person_details_picture"
+        android:layout_width="match_parent"
+        android:layout_height="200dp"
+        android:src="@drawable/thumb200"
+        android:scaleType="center"
+    />
+
+    <android.support.v4.widget.Space
+        android:layout_width="match_parent"
+        android:layout_height="30dp"/>
+
+    <TextView
+        android:id="@+id/person_details_name"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:textStyle="bold"
+        android:text="EDUARDO FLORES"
```

Comment:
Don't hardcode text in TextView elements in your layout.  Use the tools namespace to put sample text that is only visible in the IDE while designing.  Also, don't use your name as the sample text, try using something more like "John Doe" or historical names like "George Washington".

tools:text="John Doe"
instead of 
android:text="EDUARDO FLORES"